documentation/book/con-kafka-mirror-maker-authentication.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-mirror-maker-authentication.adoc
+
+[id='con-kafka-mirror-maker-authentication{context}']
+= Authentication support in Kafka Mirror Maker
+
+Authentication can be configured in the `authentication` sub-property of `consumer` and `producer` properties in `KafkaMirrorMaker.spec`.
+The `authentication` property specifies the type of the authentication mechanisms which should be used and additional configuration details depending on the mechanism.
```

Comment:
Since I hassled @scholzj about this earlier... can we retain the word "mechanism" for the SASL sense of that word and use "method" when talking about the more abstracted choice between TLS and SASL?